default.json

```diff
@@ -161,7 +161,7 @@
       "name": "German Store",
       "url": "/de",
       "elasticsearch": {
-        "host": "localhost:8080/api/catalog",
+        "host": "SERVER_HOST:SERVER_PORT/api/catalog",
```

Comment:
Please use complete url instead

Reply:
this points to the hostname as it does in https://github.com/EdwinBetanc0urt/proxy-adempiere-api-docker/blob/bugfix/redis-connection/default.json#L3